test/implementations/premint.ts

```diff
@@ -0,0 +1,155 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import {
+  ADDRESS_ZERO,
+  mintFromImplErc20Pay,
+  nestMintFromImplErc20Pay,
+  ONE_ETH,
+  singleFixtureWithArgs,
+} from '../utils';
+import { Contract } from 'ethers';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { ERC20Mock } from '../../typechain-types';
+
+async function multiAssetFixture(): Promise<Contract> {
+  return await singleFixtureWithArgs('RMRKMultiAssetImplPreMint', [
+    'MultiAsset',
+    'MR',
```

Comment:
Like I mentioned before. It would be cool to rename this to `MA`, for someone taking a look at our codebase, who is unfamiliar with the MultiResource, this could be confusing 🤔 

```suggestion
    'MA',
```

Reply:
I'd suggest doing this for all instances.

Reply:
Will do on dev, thanks!